compiler/luci/service/src/Nodes/CircleRsqrt.cpp

```diff
@@ -24,4 +27,15 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CircleRsqrt *)
   return _graph->nodes()->create<luci::CircleRsqrt>();
 }
 
+namespace sinf
+{
+
+loco::TensorShape Algorithm::visit(const luci::CircleRsqrt *node)
+{
+  const auto input_shape = circle_shape(loco::must_cast<CircleNode *>(node->x()));
```

Comment:
```suggestion
  const auto input_x = loco::must_cast<CircleNode *>(node->x());
  const auto input_shape = circle_shape(input_x);
```

Reply:
The code is easier to read after splitting it into casting and getting the shape. I changed that. Thank you!